tests/e2e/src/steps/portal.steps.ts

```diff
@@ -154,18 +154,10 @@ Then("Clipboard is not empty", async function (this: ICustomWorld) {
 });
 
 Given(
-  "I fill the {string} input field by {string}",
```

Comment:
I searched in the project all uses of 
`I fill the {string} input field by {string}`
And found only a few (3) of them - that use in fact not very good-looking placeholder XPath.
`When I fill the "//*[@placeholder='Ethereum address']" input field by "clipboard"`
I think that we can still replace with new functionality to make it more understandable and good-looking.

<img width="1230" alt="image" src="https://github.com/matter-labs/dapp-portal/assets/118197764/e6ba6e44-9851-4b28-bf35-e964e543d0a7">

Reply:
Do you think it's better to roll-back?
`I fill the {string} input field on the Contacts page with {string} text`
Because we use it through application not only in Contacts page, e.g. 
<img width="894" alt="image" src="https://github.com/matter-labs/dapp-portal/assets/118197764/d9828c4f-cfd1-4e9f-8f8b-881dc0972958">

Reply:
@abilevych the idea of deleting the current one implementation is just avoiding any confusing in future as we have presented another universal approach that already includes the previous one. Another idea is to split selectorType and selectorValue for easy handling.